Airports/EGKN/Basic.txt

```diff
@@ -1,3 +1,3 @@
 Kenley
 N051.18.20.000 W000.05.44.000
-000.000
+000.000
```

Comment:
```suggestion
119.775
```
It has an A/G service: https://www.aidu.mod.uk/aip/pdf/ad/EGKN-Kenley-Textual.pdf, pg 4

Reply:
It does, on 119.760. This is, however, in no way related to this PR (this file shouldn't have been editted at all), and this Basic.txt frequency is, as far as I'm aware, not used for anything.